components/support/rc_crypto/nss/src/ec.rs

```diff
@@ -360,6 +369,7 @@ fn create_ec_params_for_curve(curve: Curve) -> Result<Vec<u8>> {
     // https://searchfox.org/mozilla-central/rev/ec489aa170b6486891cf3625717d6fa12bcd11c1/dom/crypto/WebCryptoCommon.h#299
     let curve_oid_tag = match curve {
         Curve::P256 => nss_sys::SECOidTag::SEC_OID_ANSIX962_EC_PRIME256V1,
```

Comment:
Just a nit, but I'd probably change the P256 define to SEC_OID_SECG_EC_SECP256R1 for consistency (they're the same: https://searchfox.org/mozilla-central/source/security/nss/lib/util/secoidt.h#509)